docs/advanced-topics/docker-alternatives.rst

```diff
@@ -5,15 +5,17 @@ Docker Alternatives
 ===================
 
 In some situations using Docker may be impractical because it requires all users to have root access.
-Several alternatives have been developed to make it possible to run rootless containers, including
-`Singularity <https://sylabs.io/docs/>`_ and 
-`rootless Docker <https://www.docker.com/blog/experimenting-with-rootless-docker/>`_.
 While Dockstore uses Docker by default, if necessary it may be possible to run your workflows with one
 of these alternatives. Because the call to Docker or an alternative is made by the workflow runner, usually cwltool
 or Cromwell, and not Dockstore directly, the difficulty of configuring a Docker alternative depends on the workflow
 type. Some Dockstore entries will run seamlessly without Docker, and some may be entirely incompatible in a rootless
 environment.
 
+Rootless Docker
+---------------
+`Experiments <https://medium.com/@tonistiigi/experimenting-with-rootless-docker-416c9ad8c0d6#:~:text=Docker%20engine%20provides%20lots%20of,Linux%20you%20need%20privileged%20capabilities.>`__ with a version of Docker that does not require root permissions have been proposed and experimented with for a while now. In December 2020 with the release of Docker Engine v20.10, "rootless Docker" has officially become part of Docker itself. While this does open the door for some systems, it does require special setup and it has some limitations. You can read all about it on `Docker's own docs about Rootless Mode <https://docs.docker.com/engine/security/rootless/>`__.
+
+
 Singularity
```

Comment:
Add another section on Shifter? https://shifter.readthedocs.io/en/latest/

Also, examples from Nextflow using Shifter: https://www.nextflow.io/docs/latest/shifter.html

Reply:
will be continued in https://github.com/dockstore/dockstore/issues/4978